client/src/routes/club/index.js

```diff
@@ -37,8 +38,10 @@ export default class Club extends Component {
             <Loading />
           ) : (
             <div>
-              <h2>Spielplan (Auszug)</h2>
+              <h2>Spielplan (Rückblick)</h2>
               <Schedule {...club} />
+              <h2>Spielplan (Vorschau)</h2>
+              <Preview {...clubPreview} />
```

Comment:
Whenever one of the three ajax requests resolves, the component re-renders - here you're not checking if `clubPreview` is actually resolved, which causes an exception if `club` resolved before `clubPreview`. Instead, wrap the `clubPreview` rendering in it's own block - something like this:
```js
          {pending && pending.clubPreview ? (
            <Loading />
          ) : (
            <div>
              <h2>Spielplan (Vorschau)</h2>
              <Preview {...clubPreview} />
            </div>
          )}
```